src/acl/Options.cc

```diff
@@ -224,22 +210,30 @@ Acl::OptionsParser::parse()
                 if (oex.hasValue)
                     option.configureWith(oex.value());
                 else
-                    option.configureDefault();
+                    option.configureFlag(oex.isSet());
                 break;
             }
         }
         // else skip supported parameter flag
     }
+}
+
 
-    /* hack: regex code wants to parse all -i and +i flags itself */
-    for (const auto name: flagsToSkip)
-        ConfigParser::TokenPutBack(name);
+const Acl::Options &
+Acl::CaseLineOptions::options()
+{
+    static const Acl::BooleanOption CaseInsensitiveOn;
+    static const Acl::BooleanOption CaseInsensitiveOff;
+    static const Acl::Options MyOptions = { { "-i", &CaseInsensitiveOn }, { "+i", &CaseInsensitiveOff } };
+    CaseInsensitiveOn.linkWith(&caseInsensitive);
+    CaseInsensitiveOff.linkWith(&caseInsensitive);
+    return MyOptions;
```

Comment:
I assume this code works, but it makes no sense to the reader because the obviously opposite boolean options are configured in an identical way except for the option name (not just "symmetric", but _identical_). The trick is in the code somewhere else (the reader cannot see it and most will not guess that it exists) that interprets the option _name_ prefix and assigns `+` a special meaning.

This is really confusing and, judging by the diff alone, this PR looks responsible for introducing this pattern (even though the code that treats `+` specially obviously existed prior to this PR). Would it be a lot of work to refactor this so that Acl::BooleanOption accepts an optional "turn me off" spelling of the parameter name and handles both spellings instead of using two BooleanOption objects to represent one boolean value?

```C++
     static const Acl::BooleanOption CaseInsensitive;
     static const Acl::Options MyOptions = { { &CaseInsensitive, "-i", "+i" } };
```
 
... and dropping/removing the special `+` interpreting code (as far as leading options parsing is concerned)? In the future (outside this PR scope), other options may use option name pairs like "--enable-foo" and "--disable-foo". This approach can be applied to all options, not just boolean ones if the second option spelling always triggers a "do not enable this option" or "go back to default setting" effect (by calling the dedicated Option::reset() or similar API).